docs/configuration/index.md

```diff
@@ -1249,9 +1249,9 @@ A `javascript` option is also available but should only be used for prototyping
 If an `affinityConfig` is provided (as part of `fillCapacity` and `equalDistribution` strategies) for a given task, the list of workers eligible to be assigned is determined as follows:
 
 - a non-affinity worker if no affinity is specified for that datasource. Any worker not listed in the `affinityConfig` is considered a non-affinity worker.
-- a non-affinity worker if preferred workers are not available and the affinity is _weak_ i.e. `strong: false`.
+- a non-affinity worker if preferred workers are not available and the affinity is *weak* i.e. `strong: false`.
```

Comment:
Since we use the google style [docs](https://github.com/apache/druid/blob/master/docs/development/docs-contribute.md#style-guide), it's [recommended](https://developers.google.com/style/text-formatting) to use `_` over `*` for italics:
> Use italics formatting, <i> or _, when drawing attention to a specific word or phrase, such as when [defining terms](https://developers.google.com/style/formatting-key-terms) or [using words as words](https://developers.google.com/style/formatting-words-as-words).
Although an asterisk, *, can also indicate italics in Markdown, we recommend underscores to make it easier for humans to distinguish italics from bold in the Markdown file.